datalad/downloaders/tests/test_http.py

```diff
@@ -123,26 +126,46 @@ def enter_new(self):
         # pop last used credential, so we would use "new" ones
         del self._fixed_credentials[0]
 
+
```

Comment:
when i `git grep skip_if` it's not there.  there are those other `skip_if_something` functions, but not the simple one you wrote this morning -- the plain `skip_if`.  what am i missing here?...and i'm not quite sure what you mean by mother branch, please clarify.